packages/explorer/README.md

```diff
@@ -1,85 +1,53 @@
+Starting with MUD 2.2, the MUD templates come with the World Explorer installed.
+These instructions are how to use World Explorer on earlier versions of MUD.
 
-   ```sh
-   pnpm add @latticexyz/explorer
-   ```
-
-2. **Start a local development chain**
-
-   Ensure you have a local development chain running.
+The easiest way to get World Explorer for earlier MUD versions is to create a project with the new template.
 
-3. **Run the World Explorer**
+1. Create a project with the new template, which has World Explorer.
+   Then, delete the files you no longer need.
 
-   ```sh
-   npx @latticexyz/explorer --worldAddress <YOUR_WORLD_ADDRESS>
+   ```sh copy
+   pnpm create mud@main explorer --template vanilla
+   cd explorer
+   rm -rf packages/client packages/contracts/[st]*
    ```
 
-   Alternatively, if you have a worlds configuration file:
-
-   ```sh
-   npx @latticexyz/explorer --worldsConfigPath <PATH_TO_WORLDS_CONFIG>
+1. Edit `mprocs.yaml` to remove the definitions for `client`, `contracts`, and `anvil`.
+
+   ```yaml filename="mprocs.yaml" copy
+   procs:
+     indexer:
+       cwd: packages/contracts
+       shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
+       env:
+         RPC_HTTP_URL: "http://127.0.0.1:8545"
+         FOLLOW_BLOCK_TAG: "latest"
+         SQLITE_FILENAME: "indexer.db"
+     explorer:
+       cwd: packages/contracts
+       shell: pnpm explorer
    ```
 
-   Note: You can use `@latticexyz/store-indexer` for indexing your world's data.
-
-## CLI arguments
-
-The World Explorer accepts the following CLI arguments:
-
-| Argument          | Description                                                                      | Default value |
-| ----------------- | -------------------------------------------------------------------------------- | ------------- |
-| `worldAddress`    | The address of the world to explore                                              | None          |
-| `worldsFile`      | Path to a worlds configuration file (used to resolve world address)              | None          |
-| `indexerDatabase` | Path to your SQLite indexer database                                             | "indexer.db"  |
-| `chainId`         | The chain ID of the network                                                      | 31337         |
-| `port`            | The port on which to run the World Explorer                                      | 13690         |
-| `env`             | The environment to run the World Explorer in (e.g., "development", "production") | "production"  |
-
-## Example setup
-
-An example setup is provided in the `examples/local-explorer` directory, demonstrating a full setup for using the World Explorer in a local development environment:
-
-1. **Setup**
-
-   ```sh
-   cd examples/local-explorer && pnpm install
-   ```
-
-2. **Run**
-
-   ```sh
-   pnpm dev
-   ```
-
-   This command starts all necessary processes, including a local chain, indexer, and the explorer.
-
-## Contributing
```

Comment:
I'd prefer not to remove this section, seems relevant to be included in README in my opinion.

Reply:
If we have almost everything in the doc in the README, then we have two files we need to keep in sync. How about we have a link in the README?

Reply:
I undid the README changes here, and created a separate PR #3093 . Let's improve the docs where we can in this PR, and for now leave the README as it is. I think it's still important to have it in README even if it means keeping two files in sync. We can iterate on that later and perhaps find some kind of solution.